packages/stegoveritas/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
+pkgver=1.1
+pkgrel=1
+pkgdesc='Automatic image steganography analysis tool'
+arch=('any')
+groups=('blackarch' 'blackarch-stego')
+url='https://github.com/bannsec/stegoVeritas/releases'
```

Comment:
Please, remove the "releases" string.

Reply:
So maybe the description `URL pointing to download section of package` from the template should be change to `URL pointing to the source code or download section of package` https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python